docs/v0.21/testing-0.21rc2.md

```diff
@@ -1,43 +1,44 @@
 
 **What's wrong with the current (legacy) wallets?**
 
-The current wallet was designed at a time when what Bitcoin could be used for was not yet fully understood. This led to a wallet design language that focused on maintaining a collection of [private keys](https://en.bitcoin.it/wiki/Private_key). As Bitcoin has progressed, this design language has held back the wallet from fully utilizing the expressiveness of [Bitcoin Script](https://en.bitcoin.it/wiki/Script). New features have had to be hacked on to the wallet.
+The current wallet was designed when what Bitcoin could be used for was not yet fully understood. This led to a wallet design language that focused on maintaining a collection of [private keys](https://en.bitcoin.it/wiki/Private_key). As Bitcoin has progressed, this design language has held back the wallet from fully utilizing the expressiveness of [Bitcoin Script](https://en.bitcoin.it/wiki/Script). New features have had to be hacked on to the wallet.
 
 **Why the Switch to SQlite?**
 
 As mentioned; The current wallet uses `BerkelyDB 4.8`, which is 10 years old. This database is not actively maintained, not meant to be used as an application database, and is susceptible to file corruptions. Since the move to descriptor wallets introducing breaking compatibility changes,
 
-SQlite was chosen as a new database because it provides certain guarantees that are important for ensuring that the wallet remains backwards compatible moving forward. Furthermore, unlike `BerkelyDB 4.8`, SQlite allows us to have a one file wallet instead of a wallet directory.
+SQLite was chosen as a new database because it provides certain guarantees necessary for ensuring that the wallet remains backwards compatible moving forward. Furthermore, unlike `BerkelyDB 4.8`, SQlite allows us to have a one file wallet instead of a wallet directory.
 
 ### 1. Preparation
-If you grabbed the binary for this release candidate, you're good to go. If you went down the source route, it is required that you installed the `sqlite3` dependency and compiled the source code with wallet functionality. Skip this section if you intentionality do not want wallet functionality, or don't want to test it. This guide will be split between using the GUI and the CLI. Follow whichever you are comfortable using.
+If you grabbed the binary for this release candidate, you're good to go. If you went down the source route, it is required that you installed the `sqlite3` dependency and compiled the source code with wallet functionality. Skip this section if you intentionally do not want wallet functionality or don't want to test it. This guide will be split between using the GUI and the CLI. Follow whichever you are comfortable using.
 
 ### 2. Manual Testing
 
```

Comment:
Maybe a note on to trigger QT if compiling from source. Not everyone realizes there is a GUI. 😉

Reply:
addressed in 272bb8d46bfc4ad2b05e12f735f1a4d0fbfdd903, gave co-author credit